synapse/lib/iq.py

```diff
@@ -528,6 +531,44 @@ def addTstForms(core):
         core.addDataModel('tst', modl)
         core.addTufoProp('inet:fqdn', 'inctest', ptype='int', defval=0)
 
+    @contextlib.contextmanager
+    def getAxonCore(self):
+        with self.getTestDir() as dirn:
```

Comment:
needs docstrings and a note that its a potentially expensive helper to run because rsa key generation :D